lua/tardis/sh_interiors.lua

```diff
@@ -89,6 +91,13 @@ function TARDIS:SetupMetadata(id)
 end
 
 function TARDIS:CreateInteriorMetadata(id, ent)
+    if id == nil then
+        local cv_id = GetConVar("tardis2_create_tool_id"):GetString()
```

Comment:
I think given how this works we should name the convar something a little more generic and then it could be used by other things outside the addon potentially too. Maybe `tardis2_default_interior` or similar

Reply:
Renamed, I hope `tardis2_selected_interior` is fine